geometry_tutorials/package.xml

```diff
@@ -1,21 +1,19 @@
-<package>
```

Comment:
You need to modify the CMakelists.txt, [here](https://github.com/ros2/geometry2/blob/ros2/geometry2/CMakeLists.txt) you can find a reference.

Reply:
Thanks Alex! Everything hopefully should be fixed now